libbeat/processors/add_cloud_metadata/docs/add_cloud_metadata.asciidoc

```diff
@@ -159,7 +164,8 @@ _Azure Virtual Machine_
     "instance.id": "04ab04c3-63de-4709-a9f9-9ab8c0411d5e",
     "instance.name": "test-az-vm",
     "machine.type": "Standard_D3_v2",
-    "region": "eastus2"
+    "region": "eastus2",
+    "resourceGroup.name": "/subscriptions/641ebacb-7743-41e7-b4fa-af1167351a61/resourcegroups/testgroup/providers/Microsoft.ContainerService/managedClusters/testcluster"
```

Comment:
name of the field should be `resource_group`?

Reply:
Yes. Fixing this